dapps/react-dapp-v2/src/chains/hedera.ts

```diff
@@ -0,0 +1,54 @@
+import { JsonRpcRequest } from "@walletconnect/jsonrpc-utils";
+
+import { HEDERA_LOGO_URL } from "../constants";
+import {
+  NamespaceMetadata,
+  ChainMetadata,
+  ChainRequestRender,
+  ChainsMap,
+} from "../helpers";
+
+export const HederaMetadata: NamespaceMetadata = {
+  testnet: {
+    logo: HEDERA_LOGO_URL,
+    rgb: "118, 90, 234",
+  },
+};
+
+export const HederaChainData: ChainsMap = {
+  testnet: {
+    name: "Hedera Testnet",
+    id: "hedera:testnet",
+    rpc: ["https://testnet.hashio.io/api"],
```

Comment:
can we link to the JSON-RPC spec we created for this? I know it's not merged yet, though that would be more accurate.

Maybe we can add a comment for now that links to it?

Reply:
Done. The link I added is what the url will be when the walletconnect-specs PR gets merged. Currently it 404s, but I feel like this is the best approach